lib/molecules/admin-bar/admin-bar.controller.js

```diff
@@ -0,0 +1,74 @@
+angular
+  .module('lnPatterns')
+  .controller('lnMAdminBarController', lnMAdminBarController);
+
+lnMAdminBarController.$inject = ['$log', '$rootScope', '$scope', '$cookies', '$timeout', '$element', 'lnMAdminBarService'];
+
+function lnMAdminBarController($log, $rootScope, $scope, $cookies, $timeout, $element, lnMAdminBarService) {
+  var vm = this;
+  var status = 0; //0=init, 1=loading, 2=loadad
+  var loginCookie = '';
+
+  if (vm.lnLoadFromApi) {
+    _disableAdminBar();
+    _loginCheck();
+  } else {
+    _enableAdminBar();
+  }
+
+  /////////
+
+  function _loginCheck() {
+    loginCookie = $cookies.get('wp_admin_bar_user') || '';
+
+    if (status === 0 && loginCookie !== '') {
+      _loadData();
+    } 
+
+    if ((status === 1 || status === 2) && loginCookie === '') {
+      status = 0;
+      _disableAdminBar();
+    }
+
+    $timeout(_loginCheck, 5000);
```

Comment:
Is to periodically check if the cookie exists. Basically it checks if the user has logged in to WP, eg in other tab of the browser. If the cookie is found, the bar is shown, and if the cookie is deleted, the bar is hidden.